modules/core/src/main/scala/fs2/kafka/KafkaConsumer.scala

```diff
@@ -262,8 +318,16 @@ sealed abstract class KafkaConsumer[F[_], K, V] {
     * provided by [[KafkaConsumer]], there is no need to use this.
     */
   def fiber: Fiber[F, Unit]
+
+  /**
+   * Get consumer metrics
+   * @see org.apache.kafka.clients.consumer.KafkaConsumer#metrics
+   */
+  def metrics: F[Map[MetricName, Metric]]
 }
 
+case class KafkaTopicPartitions(topicPartitions: NonEmptySet[TopicPartition])
```

Comment:
Not sure that this is necessary. Why we can't use just a simple `NonEmptySet[TopicPartition]`?

Reply:
there are two methods with signatures:

NonEmptySet[TopicPartition] and NonEmptySet[Int] - it doesnt work in scala 2.13 :(
I dont know the simpliest way to fix this problem :)

Reply:
deleted... 

there was some clouding of the mind